tcl/db_cache.tcl

```diff
@@ -141,7 +141,24 @@ proc qc::db_cache_select_table { args } {
         # Use ns_cache
         # Create the cache if it doesn't exist yet
         if { ! [in [ns_cache_names] db] } {
-	    ns_cache create db -size [expr 1024*1024] 
+            # DB cache size
+            set param_name db_cache_size
+            db_0or1row {select param_value from params where param_name=:param_name} {
```

Comment:
My concern is that this qry is run for every request which has a performance overhead.
Also we need to be able to run qcode outside Naviserver for muppet where we have no db access.